services/web/server/src/simcore_service_webserver/projects_handlers.py

```diff
@@ -0,0 +1,118 @@
+# pylint: disable=global-statement
+
+
+import json
+import logging
+
+from aiohttp import web
+
+from .login.decorators import RQT_USERID_KEY, login_required
+from .projects_fakes import Fake
+
+log = logging.getLogger(__name__)
+
+
+ANONYMOUS_UID = -1 # For testing purposes
+
+
+@login_required
+async def list_projects(request: web.Request):
+    uid = request.get(RQT_USERID_KEY, ANONYMOUS_UID)
+
+    # TODO: implement all query parameters as in https://www.ibm.com/support/knowledgecenter/en/SSCRJU_3.2.0/com.ibm.swg.im.infosphere.streams.rest.api.doc/doc/restapis-queryparms-list.html
+    start = request.match_info.get('start', 0)
```

Comment:
why you don't use extract_and_validate here? is it because it's a jsonschema?

Reply:
actually I had a similar problem with the workbench with the start_pipeline function. that's why I asked...